config/webpack/plugins/htmlRenderPlugin.js

```diff
@@ -0,0 +1,82 @@
+const { partition } = require('lodash');
+const HtmlRenderPlugin = require('html-render-webpack-plugin');
+
+const { writeStartConfig } = require('../utils/startConfig');
+const product = require('../../../lib/product');
+const {
+  isStartScript,
+  paths,
+  routes,
+  environments,
+  sites,
+  transformOutputPath,
+  defaultClientEntry
+} = require('../../../context');
+
+const createPublicUrl = (publicPath, asset) => {
+  const host = publicPath.endsWith('/') ? publicPath : `${publicPath}/`;
+
+  return `${host}${asset}`;
+};
+
+const mapStatsToParams = ({ clientStats, routeName }) => {
+  const { entrypoints } = clientStats;
+  const assets = entrypoints[routeName]
+    ? entrypoints[routeName].assets
+    : entrypoints[defaultClientEntry].assets;
+
+  const [styles, scripts] = partition(assets, asset => asset.endsWith('.css'));
```

Comment:
Did we need to guard this against other types of assets?

Reply:
I looked into this and it seems not. `file-loader` doesn't emit files to the entrypoints. I could be safer here I guess but there's no known use case.